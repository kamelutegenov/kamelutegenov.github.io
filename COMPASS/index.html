<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<title>Главная / Компасс | Переезд на ПМЖ в Болгарию</title>
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<link rel="stylesheet" href="fonts/fonts.css">
	<link rel="stylesheet" href="css/bootstrap.min.css">
	<link rel="stylesheet" href="css/slick.css">
	<link rel="stylesheet" href="css/slick-theme.css">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
	<link rel="stylesheet" href="css/jquery.arcticmodal-0.3.css">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<div class="overlay"></div>
	<header class="header" id="header">
		<div class="head">
			<div class="container">
				<div class="row justify-content-around align-items-center">
					<div class="col-4 col-sm-4 col-md-3 col-lg-2">
						<a href="index.html" class="logo">
							<img src="img/logo.png" alt="Logo">
						</a>
					</div>
					<div class="col-lg-7 mn">
						<ul class="menu">
							<li class="menu__item">
								<a href="index.html">Главная</a>
							</li>
							<li class="menu__item">
								<a href="about.html">О компании</a>
							</li>
							<li class="menu__item">
								<a href="move.html">Недвижимость и переезд</a>
							</li>
							<li class="menu__item">
								<a href="contacts.html">Контакты</a>
							</li>
						</ul>
					</div>
					<div class="col-md-7 col-lg-3 nm">
						<div class="number">
							<button class="icon_btn iicon">
								<img src="img/phone.png" alt="" class="number__icon">
							</button>
							<div class="phones">
								<h3 class="phones__title">
									+7 (495) 12-321-345
								</h3>
								<h3 class="phones__title">
									+7 (495) 12-321-345
								</h3>
							</div>
						</div>
					</div>
					<div class="col-1 col-sm-1 col-md-1 posr">
						<div class="sandwich">
						</div>
						<div class="wrapper"></div>
					</div>
				</div>
				</div>
				
			</div>
			<div class="row">
				<div class="container">
					<div class="col-md-12">
						<ul class="burger">
								<li class="burger__item"><a href="index.html">Главная</a></li>
								<hr>
								<li class="burger__item"><a href="about.html">О компании</a></li>
								<hr>
								<li class="burger__item"><a href="move.html">Недвижимость и переезд</a></li>
								<hr>
								<li class="burger__item"><a href="contacts.html">Контакты</a></li>
								<hr>
						</ul>
					</div>
				</div>
			</div>
		<h1 class="bulgaria">
			Болгария
		</h1>
		<h2 class="you">
			Ждет вас
		</h2>
		<a href="#advantages"><div class="pointer"></div></a>
		<!-- Модельное окно -->
					<div style="display: none;">
					    <div class="box-modal" id="exampleModal">
					        <div class="box-modal_close arcticmodal-close"></div>
					        <form action="#" class="form">
					        	<h4 class="form__title">
					        		Заказать обратный звонок
					        	</h4>
					        	<p class="form__text">
					        		Заполните форму и мы свяжемся с вами как можно скорее. До скорой встречи
					        	</p>
					        	<input type="text" name="text" class="form__input" placeholder="Имя">
					        	<input type="email" name="email" class="form__input" placeholder="E-mail" required>
					        	<input type="tel" name="phone" class="form__input form__input_last" placeholder="Телефон" required>
					        	<button type ="submit" class="form__btn btn">
					        		<i class="fa fa-phone"></i> Заказать звонок
					        	</button>
					        </form>
					    </div>
					</div>
	</header>

	<section class="advantages" id="advantages">
		<div class="container">
			<div class="row">
				<div class="col-lg-12">
					<h2 class="first-title">
						Наши преимущества
					</h2>
				</div>
			</div>	
			<div class="row">
				<div class="col-md-6 col-lg-4">
					<div class="box">
						<img src="img/box1.png" alt="" class="box__img">
						<h4 class="box__title">
							Надежность
						</h4>
						<div class="box__line"></div>
						<p class="box__text">
							Мы всегда придем к вам на помощь в решении самых неоднозначных вопросов
						</p>
					</div>
				</div>

				<div class="col-md-6 col-lg-4">
					<div class="box">
						<img src="img/box2.png" alt="" class="box__img">
						<h4 class="box__title">
							Достоверность
						</h4>
						<div class="box__line"></div>
						<p class="box__text">
							Достоверность - главный принцип нашей работы. Мы знаем все тонкости переезда в Болгарию и оформления соответствующих документов
						</p>
					</div>
				</div>

				<div class="col-md-12 col-lg-4">
					<div class="box box_bot">
						<img src="img/box3.png" alt="" class="box__img">
						<h4 class="box__title">
							Компетентность
						</h4>
						<div class="box__line"></div>
						<p class="box__text">
							Наши специалисты с большим опытом работы помогут разобраться во всех юридических тонкостях переезда
							на ПМЖ в Болгарию
						</p>
					</div>
				</div>
			</div>
		</div>
	</section>

	<section class="how" id="how">
		<div class="fon"></div>
		<div class="container">
			<div class="row">
				<div class="col-lg-12">
					<h2 class="first-title first-title_white">
						Переезд в Болгарию
					</h2>
				</div>
			</div>

			<div class="row">
				<div class="col-lg-12">
					<div class="djim">
						<p class="djim__text">
							«Если Вам не нравится то место, где вы находитесь - <br>
							смените его, Вы же не дерево»
						</p>

						<h5 class="djim__title">
							Джим Рон
						</h5>
					</div>
				</div>
			</div>

			<div class="row">
				<div class="col-lg-8 offset-lg-2">
					<div class="question">
						<img src="img/pointer.png" alt="" class="question__pointer question__pointer_top">
						<p class="question__text">
							Пришло время и перед Вами стоит вопрос
						</p>

						<h4 class="question__title">
							Как переехать в Болгарию на ВНЖ или ПМЖ?
						</h4>
						<img src="img/pointer.png" alt="" class="question__pointer question__pointer_bot">
					</div>
				</div>
			</div>

			<div class="row">
				<div class="col-lg-8 offset-lg-2">
					<div class="answer">
						<h2 class="answer__title">
							Мы знаем ответ
						</h2>

						<p class="answer__text">
							Подберем для Вас оптимальный вариант и предложим лучшие условия!
						</p>
					</div>
				</div>
			</div>

			<div class="row">
				<div class="col-lg-4 offset-lg-4">
					<button class="open open_wh">
						Узнать больше
					</button>
				</div>
			</div>
		</div>
	</section>

	<section class="services" id="services">
		<div class="container">
			<div class="row">
				<div class="col-lg-12">
					<h2 class="first-title">
						Недвижимость и транспортные услуги
					</h2>
				</div>
			</div>
			<div class="row">
				<div class="col-lg-8 offset-lg-2">
					<div class="houses">
						<img src="img/home-first.png" alt="City" class="houses__img houses__img_first">
						<img src="img/home-center.png" alt="City" class="houses__img houses__img_center">
						<img src="img/home-last.png" alt="City" class="houses__img houses__img_last">
						<h4 class="houses__title">
							Мечтаете о доме на Солнечном берегу?
						</h4>

						<p class="houses__text">
							Или хотите выгодно инвестировать в  заграничную недвижимость? В любом случае - подходящий вариант уже ждет Вас! Большой выбор объектов жилой и коммерческой недвижимости и профессиональный сервис позволит провести сделку с удовольствием и получить нужный результат
						</p>
					</div>
				</div>
			</div>

			<div class="row">
				<div class="col-lg-8 offset-lg-2">
					<div class="houses_bot houses">
						<img src="img/Circle.png" alt="Circle" class="houses__circle">
						<p class="houses__text">
							Если Ваши чемоданы уже собраны и Вы готовы к переезду, рады предложить Вам транспортные услуги по перевозке личный вещей в Болгарию. Для успешного размещения на новом месте жительства мы поможем перевести Ваши вещи в сохранности
						</p>
					</div>
				</div>
			</div>

			<div class="row">
				<div class="col-lg-4 offset-lg-4">
					<button class="open open_blue">Узнать больше</button>
				</div>
			</div>
		</div>
	</section>

	<section class="customers" id="customers">
		<div class="container">
			<div class="row">
				<div class="col-lg-12">
					<h2 class="first-title">
						Отзывы наших клиентов
					</h2>
				</div>
			</div>

			<div class="row">
				<div class="col-sm-10 offset-sm-1 col-lg-10 offset-lg-1">
					<div class="slider">
						<div class="carousel">
							<img src="img/photo.png" alt="Photo" class="carousel__img">
							<div class="info">
								<div class="about">
									<div class="ad">
										<div class="about__item">Диана апасова</div>
										<div class="about__item"><a href="#">vk.com/apasova</a></div>
									</div>
									<div class="about__text">Москва <i class="fa fa-arrow-right"></i> София</div>
								</div>
								<p class="info__text">
									Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmodtempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
								</p>
							</div>
						</div>

						<div class="carousel">
							<img src="img/photo.png" alt="Photo" class="carousel__img">
							<div class="info">
								<div class="about">
									<div class="about__item">Диана апасова</div>
									<div class="about__item"><a href="#">vk.com/apasova</a></div>
									<div class="about__text">Москва <i class="fa fa-arrow-right"></i> София</div>
								</div>
								<p class="info__text">
									Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
								</p>
							</div>
						</div>
					</div>
				</div>
			</div>

			<div class="row">
				<div class="col-lg-8 offset-lg-2">
					<div class="br-bot">
						
					</div>
				</div>
			</div>
		</div>
	</section>

	<section class="consultation" id="consultation">
		<div class="container">
			<div class="row">
				<div class="col-lg-12">
					<div class="first">		
						<h2 class="first__title first-title">
							Есть вопросы<br>
							по переезду в болгарию?
						</h2>
						<p class="first__text">
							Мы проконсультируем<br>
							Вас бесплатно!
						</p>

						<img src="img/pointer.png" alt="" class="first__img">
					</div>
				</div>
			</div>

			<div class="row">
				<div class="col-sm-12 col-md-6 offset-md-3 justify-content-center col-lg-4 offset-lg-4">
					<form class="form">
						<input type="text" placeholder="ИМЯ" required class="form__item">
						<input type="email" placeholder="E-MAIL" required class="form__item">
						<input type="tel" placeholder="ТЕЛЕФОН" required class="form__item">

						<input type="submit" value="ЗАПИСАТЬСЯ" class="form__item form__item_sub">
					</form>
				</div>
			</div>
		</div>
	</section>

	<footer class="footer" id="footer">
		<div class="container">
			<div class="row">
				<div class="col-lg-12">
					<div class="block">
						<a href="index.html" class="block__logo">
							<img src="img/logo.png" alt="logo">
						</a>
						<p class="block__text">
							2016 / Все права защищены                                 
						</p>
						<p class="block__item">
							Политика конфиденциальности
						</p>

						<div class="num-wh">
							<button class="icon_btn">
								<img src="img/phone-wh.png" alt="" class="num-wh__img">
							</button>
							<div class="phone-wh">
								<h3 class="phone-wh__title">
									+7 (495) 12-321-345
								</h3>
								<h3 class="phone-wh__title">
									+7 (495) 12-321-345
								</h3>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</footer>

	<script src="js/jquery-3.3.1.min.js"></script>
	<script src="js/slick.min.js"></script>
	<script src="js/jquery.arcticmodal-0.3.min.js"></script>
	<script src="js/main.js"></script>
</body>
</html>