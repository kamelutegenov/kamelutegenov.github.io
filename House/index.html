<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Жилой дом</title>
	<link rel="stylesheet" href="css/normalize.css">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
	<link href="https://fonts.googleapis.com/css?family=Roboto+Slab:300,400,700&amp;subset=cyrillic" rel="stylesheet">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<header class="header" id="header">
		<div class="container">
			<div class="block">
				<div class="bl-logo">
					<img src="img/logo.png" alt="Logo" class="bl-logo__logo">
					<div class="offer">
						<h4 class="offer__title">
							Жилой дом
						</h4>

						<p class="offer__text">
							Ярославская область
						</p>
					</div>
				</div>
				<ul class="menu">
					<li class="menu__item"><a href="#">Инфраструктура</a></li>
					<li class="menu__item"><a href="#">Планировки и цены</a></li>
					<li class="menu__item"><a href="#">Материалы и отделка</a></li>
					<li class="menu__item"><a href="#">Документы</a></li>
					<li class="menu__item"><a href="#">Контакты</a></li>
				</ul>
				<div class="phone">
					<h3 class="phone__number">
						+7 (495) 544-23-12
					</h3>
					<button class="phone__btn">
						<img src="img/btn.png" alt="button" class="phone__img">
					</button>
				</div>
			</div>
		</div>
	</header>

	<section class="record" id="record">
		<div class="container">
			<div class="head">
				<h3 class="head__intro position">
					Дом готов, <span class="head__light">идёт внутренняя отделка. Срок сдачи:</span> Февраль 2016
				</h3>
				<h1 class="head__title position">
					Ваша квартира в новострое ждет вас
				</h1>
				<h3 class="head__text position">
					<span class="head__light">Уютные квартиры в</span> центре города и в 5 минутах от р. Волга
				</h3>
				<img src="img/cloud.png" alt="Cloud" class="head__img">
			</div>

			<div class="violet">
				<h3 class="violet__title">
					Запишитесь на просмотр прямо сейчас!
				</h3>
				<p class="violet__text">
					Покажем Вам квартиры в удобное для Вас время
				</p>
				<form action="#" class="form">
					<input type="text" class="form__name form__name_style" placeholder=".">
					<input type="tel" class="form__number form__name_style" placeholder=".">
					<input type="submit" class="violet__btn" value="Записаться на просмотр!">
				</form>
				<p class="violet__desc">
					Осталось всего <span class="violet__border">5</span> квартир!
				</p>
			</div>
		</div>
	</section>

	<section class="place" id="place">
		<div class="container">
			<div class="circles">
				<img src="img/circle1.png" alt="circles" class="circles__images circles__images_pos1">
				<img src="img/circle2.png" alt="circles" class="circles__images circles__images_pos2">
				<img src="img/circle3.png" alt="circles" class="circles__images">
				<img src="img/circle4.png" alt="circles" class="circles__images">
				<img src="img/circle5.png" alt="circles" class="circles__images circles__images_pos2">
				<img src="img/circle6.png" alt="circles" class="circles__images circles__images_pos1">
			</div>
		</div>
	</section>		

	<section class="term" id="term">
		<div class="container">
			<div class="ready">
				<h4 class="ready__title">
					  Дом готов. Идёт внутренняя отделка 
				</h4>
				
				<p class="ready__text">
					Срок сдачи: Февраль 2016
				</p>
			</div>
			<img src="img/like.png" alt="Like" class="like">
		</div>
	</section>	

	<section class="location" id="location">
		<div class="container">
			<div class="ideal">
				<h2 class="ideal__title">
					ИДЕАЛЬНОЕ РАСПОЛОЖЕНИЕ ДОМА
				</h2>

				<p class="ideal__text">
					Тихий и экологически чистый район
				</p>
			</div>

			<div class="images">
				<div class="river">
					<img src="img/photo1.png" alt="River" class="river__img">
					<h5 class="river__title">
						Близость к реке
					</h5>

					<p class="river__text">
						До Волги 300 метров
					</p>
				</div>

				<div class="river river_bot">
					<img src="img/photo2.png" alt="River" class="river__img">
					<h5 class="river__title">
						Зеленые зоны
					</h5>

					<p class="river__text">
						Березовая роща и сосновый бор
					</p>
				</div>

				<div class="river">
					<img src="img/photo3.png" alt="River" class="river__img">
					<h5 class="river__title">
						Соседи и атмосфера
					</h5>

					<p class="river__text river__text_w ">
						Коттеджи и деревенская
							застройка вокруг
					</p>
				</div>
			</div>
		</div>
	</section>
	
	<section class="infrastructure" id="infrastructure">
		<div class="container">
			<h3 class="block-title">
				РАЗВИТАЯ ИНФРАСТРУКТУРА
			</h3>

			<div class="objects">
				<div class="object-dev">
					<h3 class="object-dev__title">
						Социальные объекты
					</h3>

					<p class="object-dev__text">
						Волга: 350м, центральный пляж: 900м
					</p>

					<p class="object-dev__text">
						Авто парковка 5м.
					</p>

					<p class="object-dev__text">
						Игровая детская площадка 2м.
					</p>

					<p class="object-dev__text">
						4 продуктовых магазина в пределах 500м.
					</p>

					<p class="object-dev__text">
						Центр города: 4км.
					</p>

					<p class="object-dev__text">
						Почта 150007: 600м.
					</p>
				</div>

				<div class="object-dev">
					<h3 class="object-dev__title">
						Образовательные объекты
					</h3>

					<p class="object-dev__text">
						Детский сад №173   300м.
					</p>

					<p class="object-dev__text">
						Школа №46 900м.
					</p>

					<p class="object-dev__text">
						Школа №50  1300м.
					</p>

					<p class="object-dev__text">
						Гимназия №3 2400м
					</p>
				</div>

				<div class="object-dev">
					<h3 class="object-dev__title">
						Транспортные остановки
					</h3>

					<p class="object-dev__text">
						Союзная: 300м
					</p>

					<p class="object-dev__text">
						К-тр.Дружба: 600м
					</p>

					<p class="object-dev__text">
						Школа 50: 1100м.
					</p>
				</div>
			</div>	
		</div>
	</section>

	<section class="icons" id="icons">
		<div class="container">
			<div class="green-wave">
				<div class="all">
					<img src="img/clock.png" alt="clock" class="all__img">
					<h5 class="all__title">
						До центра 5 минут на машине
						<span class="all__text">или общ. транспорте</span>
					</h5>
				</div>

				<div class="all">
					<img src="img/people.png" alt="people" class="all__img">
					<h5 class="all__title">
						3 школы и детский садик
						<span class="all__text">в  шаговой доступности</span>
					</h5>
				</div>

				<div class="all">
					<img src="img/dom.png" alt="dom" class="all__img">
					<h5 class="all__title">
						Вся необходимая инфрастуктура
						<span class="all__text">продумана и обеспечена</span>
					</h5>
				</div>
			</div>
		</div>
	</section>

	<section class="design" id="design">
		<div class="container">
			<h1 class="home-title">
				ПРОДУМАННАЯ КОНСТРУКЦИЯ ДОМА
			</h1>

			<div class="home-block">
				<div class="about">
					<h3 class="about__title">
						Заголовок про стену
					</h3>

					<img src="img/wall1.png" alt="" class="about__img">

					<p class="about__text">
						Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took 
					</p>
				</div>

				<div class="about">
					<h3 class="about__title">
						Заголовок про кровлю
					</h3>

					<img src="img/wall2.png" alt="" class="about__img">

					<p class="about__text">
						Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make  
					</p>
				</div>

				<div class="about">
					<h3 class="about__title">
						Заголовок про канализацию
					</h3>

					<img src="img/wall3.png" alt="" class="about__img">

					<p class="about__text about__text_ml">
						Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever 
					</p>
				</div>
			</div>
		</div>
	</section>

	<section class="green-city">
		<div class="container">
			<h1 class="green-title">
				Комплектация
			</h1>

			<h3 class="in-each-title">
				В каждой квартире
			</h3>

			<div class="columns">
				<div class="check">
					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Газовый котёл: Название
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Ванна: Металлическая				
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Умывальник: Керамический
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Унитаз: Керамический						
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Газовая плита: Четырех конфорочная
					</p>
				</div>

				<div class="check">
					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Балкон: Застекленный
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Отделка пола: Линолеум
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Отделка пола в с/у: Плитка
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Отделка стен: Водоэмульсионная краска
					</p>
				</div>

				<div class="check">
					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Дверь входная:  Стальная
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Двери межкомнатные: Деревянные
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Окна:  ПВХ
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Отделка стен в с/у: Частично плитка керамическая
					</p>

					<p class="check__mark">
						<i class="fas fa-check-circle"></i>
						Отделка потолков: Водоэмульсионная краска
					</p>
				</div>
			</div>
		</div>
	</section>

	<section class="oreder" id="oreder">
		<div class="container">
			<div class="security">
				<h1 class="security__title">
					У НАС ЕСТЬ ВСЕ НЕОБХОДИМЫЕ ДОКУМЕНТЫ
				</h1>

				<p class="security__text">
					Обеспечивающие надежность и Вашу защиту
				</p>
			</div>

			<div class="documents">
				<img src="img/document1.png" alt="Document" class="documents__img">
				<img src="img/document2.png" alt="Document" class="documents__img">
				<img src="img/document3.png" alt="Document" class="documents__img">
				<img src="img/document4.png" alt="Document" class="documents__img">
			</div>
		</div>
	</section>

	<footer class="footer" id="footer">
		<div class="container">
			<div class="bl-footer">
				<div class="bl-logo">
					<img src="img/logo.png" alt="Logo" class="bl-logo__logo">
					<div class="offer">
						<h4 class="offer__title white">
							Жилой дом
						</h4>

						<p class="offer__text white">
							Ярославская область
						</p>
					</div>
				</div>
				
				<div class="bl-ord">
					<p class="bl-footer__text fz">
						2015 Все права защищены
					</p>
					<a href="#" class="bl-footer__link fz">
						Политика конфиденциальности
					</a>
				</div>

				<div class="address">
					<h5 class="address__title">
						<img src="img/address.png" alt="address" class="address__marker">
						Адрес офиса:
					</h5>
					<p class="address__text">
						Ярославская область<br>
						р-н Заволжский, ул.3-я Тверицкаяд.32
					</p>

					<p class="address__email">
						<span class="address__span">E-mail:</span> info@sitename.ru
					</p>
				</div>

				<div class="black">
					<h3 class="black__number">
						+7 (495) 544-23-12
					</h3>
					<button class="black__btn">
						<img src="img/black-btn.png" alt="button" class="black__img">
					</button>
				</div>
			</div>
		</div>
	</footer>
</body>
</html>