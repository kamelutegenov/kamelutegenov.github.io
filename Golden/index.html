<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

   <!--  Bootstrap CSS
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous"> -->
    <link rel="stylesheet" href="build/css/style.css">

    <title>Hello, world!</title>
  </head>
  <body>
    <header class="header" id="header">
      	<div class="container">
      		<div class="head">
	            <img src="build/img/logo.png" alt="Golden" class="logo">
	            <nav>
	              	<ul class="menu">
	                	<li class="menu__item"><a href="#">Home</a></li>
	               	 	<li class="menu__item"><a href="#">Services</a></li>
	                	<li class="menu__item"><a href="#">Portfolio</a></li>
	                	<li class="menu__item"><a href="#">About</a></li>
	                	<li class="menu__item"><a href="#">Contact</a></li>
	              	</ul>
	            </nav>
            </div>

            <div class="titles">
            	<div class="titles__first">
            		Welcome To Our Studio!
            	</div>

            	<h1 class="titles__main">
            		It’s nice to Meet you
            	</h1>

            	<button class="titles__btn">
            		Tell Me More
            	</button>
            </div>
       	</div>
    </header>
      
    <section class="types-services" id="types-services">
      <div class="container">
      	<div class="title">
      		<h2 class="title__main">
      			Services
      		</h2>

      		<p class="title__text">
      			Proin iaculis purus consequat sem cure. 
      		</p>
      	</div>

		<div class="container-services">
	      	<div class="services">
	      		<img src="build/img/icon1.png" alt="trashcan" class="services__img">
	      		<h3 class="services__title">
	      			E-Commerce
	      		</h3>
	      		<p class="services__text">
	      			Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.
	      		</p>
	      	</div>

	      	<div class="services">
	      		<img src="build/img/icon2.png" alt="laptop" class="services__img">
	      		<h3 class="services__title">
	      			Responsive Web
	      		</h3>
	      		<p class="services__text">
	      			Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.
	      		</p>
	      	</div>

	      	<div class="services">
	      		<img src="build/img/icon3.png" alt="castle" class="services__img">
	      		<h3 class="services__title">
	      			Web Security
	      		</h3>
	      		<p class="services__text">
	      			Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.
	      		</p>
	      	</div>
      	</div>
      </div>
    </section>
     
    <section class="images" id="images">
      	<div class="container">
     		<div class="title">
	      		<h2 class="title__main">
	      			Our portfolio
	      		</h2>

	      		<p class="title__text">
	      			Proin iaculis purus consequat sem cure.
	      		</p>
      		</div>

      		<div class="portfolio">
      			<img src="build/img/img1.jpg" alt="work1" class="portfolio__img">
      			<img src="build/img/img2.jpg" alt="work2" class="portfolio__img">
      			<img src="build/img/img3.jpg" alt="work3" class="portfolio__img">
      		</div>
     	</div>
    </section>
    
    <section class="about" id="about">
     	<div class="container">
      		<div class="title">
	      		<h2 class="title__main">
	      			About Us
	      		</h2>

	      		<p class="title__text">
	      			Proin iaculis purus consequat sem cure.
	      		</p>
      		</div>

      		<div class="block">
      		<div class="case">
      			<div class="content">
      				<h4 class="content__date">
      					July 2010		<br>
						Our Humble Beginnings
      				</h4>
      				<p class="content__text">
      					Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.
      				</p>
      			</div>
      			<img src="build/img/about1.jpg" alt="table" class="places">
      		</div>

      		<div class="case">
      			<div class="content left">
      				<h4 class="content__date">
      					January 2011	<br>
						Facing Startup Battles
						
      				</h4>
      				<p class="content__text">
      					Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.
      				</p>
      			</div>
      			<img src="build/img/about2.jpg" alt="coast" class="places">
      		</div>

      		<div class="case">
      			<div class="content">
      				<h4 class="content__date">
      					December 2012	<br>
						Enter The Dark Days
      				</h4>
      				<p class="content__text">
      					Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.
      				</p>
      			</div>
      			<img src="build/img/about3.jpg" alt="concert" class="places">
      		</div>

      		<div class="case">
      			<div class="content left">
      				<h4 class="content__date">
      					February 2014	<br>
						Our Triumph

      				</h4>
      				<p class="content__text">
      					Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.
      				</p>
      			</div>
      			<img src="build/img/about4.jpg" alt="аdventure" class="places">
      		</div>
      		</div>
     	</div>
    </section>

    <footer class="footer" id="footer">
      	<div class="container">
      		<div class="footer__text">
      			© Copyright 2014 FreebiesXpress.com
      		</div>
      	</div>
    </footer>
  </body>
</html>